.github/workflows/italia.yml

```diff
@@ -0,0 +1,118 @@
+name: Review PR permissions for italia/ directory
+
+on:
+  pull_request_target:
```

Comment:
This is one of the most annoying missing features in GitHub actions: workflows from forks use a `GITHUB_TOKEN` with no permission to comment on the PR and every solution is either convoluted, a foot-gun or both.

`pull_target_request` is the simplest foot-gun and is not a security issue **as long as**:
* we don't checkout of the repository's code
* there are no secrets (https://github.com/perma-id/w3id.org/settings/secrets/actions)
* changes to this file are carefully reviewed, making sure the above items don't change over time or in the proposed patch